### PR TITLE
Additional checks 360 'See all' modal

### PR DESCRIPTION
**Pull request by edugomez**
#655

### CODE REVIEW

File: cove_360/tests.py [hunk 1]
```diff
@@ -286,21 +287,21 @@
                   "on why these grants are £0 might be useful to anyone using the data, "
                   "so consider adding an explanation to the description of the grant.")},
      ['grants/0/amountAwarded'],
-     [['grants', 'Q', 2, 'Amount Awarded']]),
+     [{'sheet': 'grants', 'letter': 'Q', 'row_number': 2, 'header': 'Amount Awarded'}]),
```
Comment (edugomez) [inline cove_360/tests.py]: This has been changed from list of lists to list of dicts to be in agreement with the data structure for validation spreadsheet source path for schema validation

**Comment by edugomez** [comment]
360_see_all

Clicking 'See all' triggers the modal dialog with the location of all non-passing checks (only for >3)

**Comment by robredpath** [comment]
I'd expect the 'see all' link to be after the 'first 3 locations' - is that possible? After the prose it feels like it's going to tell me more about what the check means rather than where they are in my data.

**Comment by edugomez** [comment]
> I'd expect the 'see all' link to be after the 'first 3 locations' - is that possible?
 
Yes, that makes sense.

**Comment by edugomez** [comment]
360_see_all

360_modal

**Comment by edugomez** [comment]
@Bjwebb please review when you have a minute, I'd like to move to "done" in the non-sprint board.

**Comment by Bjwebb** [comment]
There's no "see all" link for spreadsheets, is this correct?

**Comment by edugomez** [comment]
> There's no "see all" link for spreadsheets, is this correct?

No, it should be there.

**Pull request merged**